wordle/src/Utils/gameUtils.js

```diff
@@ -40,6 +40,25 @@ const gameUtils = {
     }
     return true;
   },
+
+  indexOfLetter(letter) {
```

Comment:
There is an easier way to get the corresponding index of each letter, which is by making use of ascii code. For example, the index for A, which is 0, can be obtained by `index = letter - 'A'.`

Reply:
That's a really good point! I have committed the necessary changes to accommodate your request